app/components/Table.tsx

```diff
@@ -121,7 +121,10 @@ function Table({
       <InnerTable {...getTableProps()}>
         <thead>
           {headerGroups.map((headerGroup) => (
-            <tr {...headerGroup.getHeaderGroupProps()} key={headerGroup.id}>
+            <tr
+              {...headerGroup.getHeaderGroupProps()}
+              key={headerGroup.getHeaderGroupProps().key}
```

Comment:
This should be fine just with the spread then, guessing without the explicit `key` there are eslint errors?

Reply:
Neatest is probably to assign to a local variable and use that.

```
const { key, ...groupProps } = headerGroup.getHeaderGroupProps();
```
?

Reply:
Updated with local variable. Avoided double spread in both the destructing and in `tr`